custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/go.mod

```diff
@@ -0,0 +1,17 @@
+module prometheus-dummy-exporter
+
+go 1.20
```

Comment:
Why it is 1.20 here while it is go1.16 in Dockerfile?

Reply:
good catch